src/coreclr/vm/method.cpp

```diff
@@ -2108,7 +2108,13 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
     }
 
     if (RequiresStableEntryPoint() && !HasStableEntryPoint())
```

Comment:
```suggestion
#ifdef FEATURE_PORTABLE_ENTRYPOINTS
    return GetPortableEntryPoint();
#else    

    if (RequiresStableEntryPoint() && !HasStableEntryPoint())
```
I think this can just return portable entrypoint right away in all cases. I do not think the fine-grained ifdefs are needed